server/server.go

```diff
@@ -180,7 +176,7 @@ func (m *Command) SetupServer() error {
 		if err != nil {
 			gossipHost = m.Config.Host
 		}
-		gossipNodeSet := gossip.NewGossipNodeSet(m.Config.Host, gossipHost, gossipPort, gossipSeed, m.Server)
+		gossipNodeSet := gossip.NewGossipNodeSet(m.Config.Host, gossipHost, gossipPort, gossipSeed, m.Server, m.Server.LogOutput)
```

Comment:
I added this redundant argument because I was unsure of the significance of m.Server being passed in as a `pilosa.StatusHandler`. Looking at it now, I'd rather just change the argument type and remove the extra arg.